utils/BaseStore.js

```diff
@@ -1 +1,4 @@
-module.exports = require('dispatchr/utils/BaseStore');
+// @TODO deprecate
+//console.warn("require('fluxible/utils/createStore') is deprecated. Please use " +
```

Comment:
Got it. Makes sense. What do you think about adding the version number, where the change is expected, to the todo comments? Like: 

```js
// @TODO deprecate in v0.6.x
// console.warn("require('fluxible/utils/createStore') is deprecated...
```